packages/shared/src/components/UserBadge.tsx

```diff
@@ -0,0 +1,37 @@
+import classNames from 'classnames';
+import React, { FunctionComponent, ReactElement, ReactNode } from 'react';
+import { IconProps, IconSize } from './Icon';
+import useMedia from '../hooks/useMedia';
+import { tablet } from '../styles/media';
+
+export type UserBadgeProps = {
+  className?: string;
+  content: ReactNode;
+  Icon: FunctionComponent<IconProps>;
```

Comment:
I think it is cleaner then to put bunch of props on react element. With component we can also inject custom props more easily eg. in SquadMemberRoleBadge without cloning. It would be great for all of us to sit down one time and discuss different pros/cons since they are minor but it would be cool to standardize. 👌